webui/react/src/services/utils.ts

```diff
@@ -211,3 +211,4 @@ export const validateDetApiEnumList = (enumObject: unknown, values?: unknown[]):
 
 /* eslint-disable-next-line */
 export const noOp = (): void => {}
+export const identity = <T>(a: T): T => a;
```

Comment:
**suggestion:** under `apiConfig.ts` should we make `generateDetApi` work more like `generateApi` where  the `postProcess` field is optional so we can avoid having to pass `noOp` and `identity`?

Reply:
that seems doable and makes sense. I made some changes in how I was using postProcess. Let's leave the generateDetApi changes for another pr